test/document_test.rb

```diff
@@ -30,7 +30,7 @@ def add_to_foo key, value
 end
 
 class WithTime < Dolly::Document
-  property :created_at
+  property :created_at, default: -> {DateTime.now}
```

Comment:
@neojin I did not use `Time` as when i first added the feature nobody commented on my use of `DateTime`. I can switch it.